wsd/FileServer.cpp

```diff
@@ -527,7 +531,14 @@ void FileServerRequestHandler::handleRequest(const HTTPRequest& request,
             handleWopiRequest(request, requestDetails, message, socket);
             return;
         }
+#else
+        if (Util::startsWith(relPath, std::string("/browser/dist/remote/logo.svg")))
```

Comment:
I'd really prefer the same name lokit-extra-img.svg as we pass it through - renaming things as they go past makes things more complex to maintain I think.